### ISSUE TITLE
Cancelling BankID authentication results in error 500

### ISSUE THREAD

**Issue by oliverjohns** [bankid4keycloak/bankid4keycloak#22]
**Describe the bug**
After starting a BankID authentication via keycloak and you press cancel, `/endpoint/cancel` is called resulting in a response with error code 500, displaying the "We are sorry. An internal server error has occurred" in Keycloak.

**To Reproduce**
1. Add the BankID identity provider (Test environment)
1. Go to login page
2. Press sign in with BankID
3. Press cancel

**Expected behavior**
Be redirected back to the Keycloak login page when cancel is pressed.

**Screenshots**
![image](https://user-images.githubusercontent.com/63138155/146942142-bbd42d91-0181-4cca-9913-af7c4c54a359.png)

**Environment:**
 - OS: MacOS
 - Browser: Mozilla Firefox
 - Keycloak version: 15.0.2
 - Version of BankID 4 Keycloak: 1.0.0-SNAPSHOT
 - Java Version: 
 java version "1.8.0_301"
Java(TM) SE Runtime Environment (build 1.8.0_301-b09)
 - BankId Client: Desktop

**Additional context**
Logging in with BankID works, but cancel does not work. This is using the test environment and test BankID.

**Comment by oliverjohns** [comment]
Getting these errors in my Keycloak instance when cancelling:
```
15:05:03,639 ERROR [org.keycloak.broker.bankid.client.SimpleBankidClient] (default task-33) 
Request to /rp/v5/collect failed with status code 400 and payload 
{"errorCode":"invalidParameters","details":"No such order"}
15:32:10,885 ERROR [org.keycloak.services.error.KeycloakErrorHandler] (default task-66) 
Uncaught server error: java.lang.NullPointerException at
org.keycloak.services.managers.AuthenticationSessionManager.getUserSession(AuthenticationSessionManager.java:218)
```

**Comment by oliverjohns** [comment]
Getting these errors when cancelling: 
![image](https://user-images.githubusercontent.com/63138155/146947457-a647f4ce-dd2e-4189-a90d-fef47320bf97.png)

**Comment by oliverabclabs** [comment]
Fixed it with this change:
![image](https://user-images.githubusercontent.com/89071229/150783081-0ba9d65b-ae80-4713-9b25-1d4155077e02.png)

**Comment by ullgren** [comment]
Fixed in caee9297f0f8dd85453b7d6316477d8dc8f7fc43
